app/components/infotiles.js

```diff
@@ -2,10 +2,11 @@ import Image from "next/image";
 import Link from "next/link";
 import Styles from "../styles/Infotiles.module.css";
 
-export default function Infotiles({ data }) {
+export default function Infotiles(data) {
```

Comment:
```suggestion
export default function Infotiles(props) {
```

If you're not destructuring, better use the name props here.